package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@grapesjs/monorepo",
+  "version": "0.0.0",
+  "packageManager": "pnpm@8.6.3",
+  "scripts": {
+    "start": "pnpm --filter grapesjs start",
+    "test": "pnpm --filter grapesjs test",
+    "docs": "pnpm --filter @grapesjs/docs docs",
+    "docs:api": "pnpm --filter @grapesjs/docs docs:api",
+    "lint": "eslint .",
+    "build": "pnpm --filter \"!@grapesjs/docs\" build",
+    "clean": "find . -type d \\( -name \"node_modules\" -o -name \"build\" -o -name \"dist\" \\) -exec rm -rf {} + && rm ./pnpm-lock.yaml",
+    "format": "prettier . --write",
+    "format:check": "prettier . --check"
+  },
+  "devDependencies": {
```

Comment:
Most of the dev deps for typescript, testing, building, and lining are now in this file.